code/__DEFINES/food.dm

```diff
@@ -87,3 +87,16 @@
 #define ICE_CREAM_BLAND "bland"
 
 #define DEFAULT_MAX_ICE_CREAM_SCOOPS 3
+
+
+
+#define OVEN_SMOKE_STATE_NONE 0
```

Comment:
Is there any reason for these to be defined outside of oven file.

Reply:
guess not